R/driver-snowflake.R

```diff
@@ -35,7 +35,7 @@ getCatalogSchema <- function(conn, catalog_name = NULL, schema_name = NULL) {
 #' to aid with performance, as the SQLColumns method is more performant
 #' when restricted to a particular DB/schema.
 #' @inheritParams DBI::dbListFields
-#' @rdname Snowflake
+#' @rdname driver-Snowflake
```

Comment:
I'd be tempted to document them alltogether, adding a new section that details the snowflake method variations. But might be easiest for me to do this after the PR is merged.